arch/arm/core/aarch32/cortex_m/fault.c

```diff
@@ -1082,4 +1082,7 @@ void z_arm_fault_init(void)
 	 */
 	SCB->CCR |= SCB_CCR_STKOFHFNMIGN_Msk;
 #endif /* CONFIG_BUILTIN_STACK_GUARD */
+#ifdef CONFIG_TRAP_UNALIGNED_ACCESS
+	SCB->CCR |= SCB_CCR_UNALIGN_TRP_Msk;
+#endif /* CONFIG_TRAP_UNALIGNED_ACCESS */
```

Comment:
For ARMv6-M and ARMv8-M Baseline cores (e.g. Cortex-M0, Cortex-M23), the SCB register is read-only and `UNALIGN_TRP` always read as one (i.e. always enabled).

Reply:
Oh neat, thanks for pointing that out.